### ISSUE TITLE
Getting error while generating passcode

### ISSUE THREAD

**Issue by nilayAcquire** [twilio-labs/plugin-rtc#57]
[DEBUG] Config File: C:\Users\Nilay\.twilio-cli\config.json
[DEBUG] Using profile: nilTest
[DEBUG] -- BEGIN Twilio API Request --
[DEBUG] GET https://serverless.twilio.com/v1/Services
[DEBUG] Custom HTTP Headers:
[DEBUG] User-Agent: @twilio/cli-core/5.13.0 (twilio-node/3.53.0, node.js v14.15.1, win32 10.0.19041 x64, rtc:apps:video:deploy)
[DEBUG] -- END Twilio API Request --
[DEBUG] response.statusCode: 200
[DEBUG] response.headers: {"date":"Mon, 07 Dec 2020 11:06:05 GMT","content-type":"application/json; charset=utf-8; charset=utf-8","content-length":"1067","connection":"close","twilio-concurrent-requests":"1","twilio-request-id":"RQf8f9bb1253094b7a95af2462772442e8","twilio-request-duration":"0.024","access-control-allow-origin":"*","access-control-allow-headers":"Accept, Authorization, Content-Type, If-Match, If-Modified-Since, If-None-Match, If-Unmodified-Since","access-control-allow-methods":"GET, POST, DELETE, OPTIONS","access-control-expose-headers":"ETag","access-control-allow-credentials":"true","x-powered-by":"AT-5000","x-shenanigans":"none","x-home-region":"us1","x-api-domain":"serverless.twilio.com","strict-transport-security":"max-age=31536000"}
[DEBUG] -- BEGIN Twilio API Request --
[DEBUG] GET https://serverless.twilio.com/v1/Services/ZS98611d9174366825df226b9a3916e14e/Environments
[DEBUG] Custom HTTP Headers:
[DEBUG] User-Agent: @twilio/cli-core/5.13.0 (twilio-node/3.53.0, node.js v14.15.1, win32 10.0.19041 x64, rtc:apps:video:deploy)
[DEBUG] -- END Twilio API Request --
[DEBUG] response.statusCode: 200
[DEBUG] response.headers: {"date":"Mon, 07 Dec 2020 11:06:06 GMT","content-type":"application/json; charset=utf-8; charset=utf-8","content-length":"380","connection":"close","twilio-concurrent-requests":"1","twilio-request-id":"RQ80af8255e73343dd85d0b6e8abf82971","twilio-request-duration":"0.016","access-control-allow-origin":"*","access-control-allow-headers":"Accept, Authorization, Content-Type, If-Match, If-Modified-Since, If-None-Match, If-Unmodified-Since","access-control-allow-methods":"GET, POST, DELETE, OPTIONS","access-control-expose-headers":"ETag","access-control-allow-credentials":"true","x-powered-by":"AT-5000","x-shenanigans":"none","x-home-region":"us1","x-api-domain":"serverless.twilio.com","strict-transport-security":"max-age=31536000"}
[DEBUG] Found command "rtc:apps:video:deploy" plugin: @twilio-labs/plugin-rtc
 » twilio-cli encountered an unexpected error. To report this issue, execute the command with the "-l debug" flag, then copy the output to a new issue here: "https://github.com/twilio-labs/plugin-rtc"
[DEBUG] Cannot read property 'sid' of undefined
[DEBUG] TypeError: Cannot read property 'sid' of undefined
    at DeployCommand.getAppInfo (C:\Users\Nilay\.twilio-cli\node_modules\@twilio-labs\plugin-rtc\src\helpers.js:78:64)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
    at async DeployCommand.run (C:\Users\Nilay\.twilio-cli\node_modules\@twilio-labs\plugin-rtc\src\commands\rtc\apps\video\deploy.js:18:20)
    at async DeployCommand._run (C:\Users\Nilay\.twilio-cli\node_modules\@oclif\command\lib\command.js:43:20)
    at async Config.runCommand (C:\Users\Nilay\AppData\Roaming\npm\node_modules\twilio-cli\node_modules\@oclif\config\lib\config.js:173:24)
    at async Main.run (C:\Users\Nilay\AppData\Roaming\npm\node_modules\twilio-cli\node_modules\@oclif\command\lib\main.js:27:9)
    at async Main._run (C:\Users\Nilay\AppData\Roaming\npm\node_modules\twilio-cli\node_modules\@oclif\command\lib\command.js:43:20)
 »   Error: EEXIT: 1
 »   Code: EEXIT

**Comment by timmydoza** [comment]
Thanks for the report @nilayAcquire 

Could you please provide some additional context? Specifically, it would be helpful if you could provide the following information:

- What steps did you take that produced this error?
- What version of the Twilio CLI are you using?
- What version of plugin-rtc are you using?
- What OS and version are you running?

In the meantime, let's try upgrading your version of the plugin to see if that does the trick. 

To upgrade your CLI plugins, run `twilio plugins:update`.

Once that is done, run `twilio rtc:apps:video:delete` to remove any video apps that have been deployed with an older version of the plugin. 

Then, try deploying a new video app. I hope this helps!

**Comment by aaalaniz** [comment]
Hey @nilayAcquire 

I am closing this issue for now. Please let us know if you need any further assistance triaging this issue.

Thanks!
